tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,362 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "T_eff = 4780  # temperature of hatp11[K], Bakos et al. 2010\n",
+    "\n",
+    "# Use synphot.models and synphot.spectrum module to generate a blackbody spectrum\n",
+    "photlam_hatp11 = SourceSpectrum(BlackBody1D, temperature=T_eff)\n",
+    "\n",
+    "# Compute the flux at Earth from the luminosity \n",
+    "R = 0.683 * u.R_sun  # radius in solar units, Deming et al. 2011\n",
+    "d = (123 * u.lyr).to(u.R_sun)  # distance to HATP11 in solar radii, from Gaia parallax\n",
+    "photlam_hatp11 = photlam_hatp11 * (R / d) ** 2  # in photon s-1 cm^-2 AA^-1\n",
```

Comment:
To avoid the explicit unit conversion that you did for `d`, simple define `d = 123 * u.lyr`. Then in the next line, call `photlam_hatp11 * float(R / d) ** 2`. The `float` call will resolve the unit differences and return the float conversion factor.